### PR TITLE
Update for Microsoft build of OpenJDK 11, Jan 2022 PSU

### PR DESCRIPTION
**Pull request by d3r3kk**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.
-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44369)

### CODE REVIEW

File: manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+InstallerSwitches:
+  Custom: ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-11.0.14.9.1-windows-x64.msi#winget
+  InstallerSha256: 9932E0088AF6107297AFFEAC0560C9FE6D04DE3D041B50D1C1AABE7C8006C189
+  ProductCode: '{D2FB917E-4B7C-47AD-837D-979D31A333E1}'
+- Architecture: arm64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-11.0.14.9.1-windows-aarch64.msi#winget
```
Comment (Trenly) [inline manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml:22]:
```suggestion
  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/70109184-0b6e-430d-8e8e-35e8e65fe8af/bb4dceb4980d266126440c159963e8e4/microsoft-jdk-11.0.14.9.1-windows-aarch64.msi
```

File: manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-11.0.14.9.1-windows-x64.msi#winget
```
Comment (Trenly) [inline manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml:18]:
```suggestion
  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/70109184-0b6e-430d-8e8e-35e8e65fe8af/e83bc9e615401db2e3e0b3c5c8fb31a3/microsoft-jdk-11.0.14.9.1-windows-x64.msi

```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml (hunk 1 above)
Comment (d3r3kk) [reply manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml:22]: Leaving, we use the aka.ms links to monitor usage.

File: manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml (hunk 2 above)
Comment (d3r3kk) [reply manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml:18]: Leaving, we use aka.ms/ links to verify usage.

Comment (Trenly) [reply manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml:18]: > Leaving, we use aka.ms/ links to verify usage.

Previous versions use the direct links.

**Review by OfficialEsco — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**